pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -250,6 +251,7 @@ func ValidateMutatingWebhookConfiguration(e *admissionregistration.MutatingWebho
 
 type validationOptions struct {
 	ignoreMatchConditions                   bool
+	matchConditionsHasParameters            bool
```

Comment:
nit: `allowParamsInMatchConditions` might be clearer. it's hard to tell if this means matchConditions _are using_ a `params` field or if `matchConditions` are _allowed_ to use a `params` var.

Reply:
ack